src/raghvendra/BrokenLinkFileUpload.java

```diff
@@ -0,0 +1,70 @@
+package raghvendra;
+import java.awt.AWTException;
+import java.awt.Robot;
+import java.awt.Toolkit;
+import java.awt.datatransfer.StringSelection;
+import java.awt.event.KeyEvent;
+import java.io.IOException;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+import org.openqa.selenium.By;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import raghvendra.base.ConstantAction;
+//Assignment-12: Perform broken link & File uploading scenario
+public class BrokenLinkFileUpload {
+	WebDriver driver=ConstantAction.start("file:///C:/Java%20Selenium/Oct22/SeleniumTechnoOct2020/resources/forms/Broken_Link_Form.html");
```

Comment:
Do not launch browser code from the variable, call the method from the main method only